lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -843,6 +847,7 @@ def create_parameters(self):
             mac_address='mac_address',
             labels='labels',
             stop_signal='stop_signal',
+            stop_timeout='stop_timeout',
```

Comment:
Support for `stop_timeout` has only been added in `docker` 2.1.0 (see https://docker-py.readthedocs.io/en/stable/change-log.html). Since we support older versions, you need to add some more code which handles that:
  1) make sure this is only inserted into `create_params` when the `docker` client library version is high enough;
  2) raise an error if this parameter has a non-trivial value and version is too low.
Check the code of `AnsibleDockerClientContainer` (below in this file) for examples on how this can be done.